tests/impl.cpp

```diff
@@ -350,10 +350,34 @@ static inline double bankersRounding(double val)
     return ret;
 }
 
-static float ranf(void)
-{
-    uint32_t ir = rand() & 0x7FFF;
-    return (float) ir * (1.0f / 32768.0f);
+/*  Written in 2015 by Sebastiano Vigna (vigna@acm.org)
```

Comment:
Since the code is released as public domain, you can simply skip the copyright notice. Simply mention the name as https://github.com/microsoft/mimalloc/blob/master/include/mimalloc/internal.h#L768

Reply:
Done.